src/browser/modules/Stream/CypherFrame/CypherFrame.tsx

```diff
@@ -190,6 +190,8 @@ export class CypherFrame extends Component<CypherFrameProps, CypherFrameState> {
       { name: 'JSON', download: this.exportJSON }
     ]
     const downloadGraphics = [
```

Comment:
This list is meant for the graphical / image downloads and I would like to keep this list with only those two items. To solve the problem of showing the text downloads I think it's better if you extend the `textDownloadEnabled` to say that text download is enabled for when the visualization is open too!

```ts
    const textDownloadEnabled = () =>
      this.getRecords().length > 0 &&
      this.state.openView &&
      [ViewTypes.TEXT, ViewTypes.TABLE, ViewTypes.CODE, ViewTypes.VISUALIZATION].includes(
        this.state.openView
      )
```

I think that should do the trick :)